### ISSUE TITLE
How did you build this?

### ISSUE THREAD

**Issue by Stay1444** [Nikilites/nuzu#6]
I'm trying to build this with a yuzu fork I made yesterday. I had to download yuzu's mbedtls fork from the wayback machine, but I didn't manage to build it because I get a linking error.

```
[12/106] Linking CXX executable bin/yuzu-cmd
FAILED: bin/yuzu-cmd 

mold: error: undefined symbol: mbedtls_cipher_cmac
>>> referenced by key_manager.cpp
>>>               src/core/libcore.a(src/core/CMakeFiles/core.dir/crypto/key_manager.cpp.o):(Core::Crypto::KeyManager::DeriveBase())
collect2: error: ld returned 1 exit status
[13/106] Building CXX object src/yuzu/CMakeFiles/yuzu.dir/cmake_pch.hxx.gch
ninja: build stopped: subcommand failed.
```

I Also tried with upstream mbedtls but I get some function errors probably because it does not match the version used in Yuzu, which was quite old.

I see that you have created a build in the releases page. Did you build that or are you just re uploading Yuzu's latest builds? I don't see mbedtls in your submodules.

**Comment by borntohonk** [comment]
> I'm trying to build this with a yuzu fork I made yesterday. I had to download yuzu's mbedtls fork from the wayback machine, but I didn't manage to build it because I get a linking error.
> 
> ```
> [12/106] Linking CXX executable bin/yuzu-cmd
> FAILED: bin/yuzu-cmd 
> 
> mold: error: undefined symbol: mbedtls_cipher_cmac
> >>> referenced by key_manager.cpp
> >>>               src/core/libcore.a(src/core/CMakeFiles/core.dir/crypto/key_manager.cpp.o):(Core::Crypto::KeyManager::DeriveBase())
> collect2: error: ld returned 1 exit status
> [13/106] Building CXX object src/yuzu/CMakeFiles/yuzu.dir/cmake_pch.hxx.gch
> ninja: build stopped: subcommand failed.
> ```
> 
> I Also tried with upstream mbedtls but I get some function errors probably because it does not match the version used in Yuzu, which was quite old.
> 
> I see that you have created a build in the releases page. Did you build that or are you just re uploading Yuzu's latest builds? I don't see mbedtls in your submodules.

All he did was rename the early access release archive name to "Windows-Nuzu.YUZU.-EA-4176", the yuzu binaries contained in the release archive are all dated 1st of march 2024

**Comment by borntohonk** [comment]
--version returns yuzu HEAD early-access-0-2518-6131-g72ac0b9787

**Comment by MarkDPierce** [comment]
They kinda deleted all the workflow files meant to help in this process. This is beyond stupid to be fair. If they knew what was up they would have corrected them or left them to be fixed. But im certain this person has never contributed to opensource let alone maintained a project.

**Comment by Stay1444** [comment]
Yeah I noticed. I have a clone with history of everything over https://git.zaroz.cloud/nintendo-back-up, but I can't manage to build it.
